lib/url.c

```diff
@@ -293,6 +296,10 @@ void Curl_freeset(struct Curl_easy *data)
   data->change.url = NULL;
 
   Curl_mime_cleanpart(&data->set.mimepost);
+
+#if defined(USE_HSTS)
```

Comment:
I think you should stick with plain `#ifdef` when there's just one define to check, as that's what we usually do in the code.